BioCro/DARPA/biocro_biomass_clean_darpa.Rmd

```diff
@@ -0,0 +1,235 @@
+---
+title: "Setaria Biomass Data Cleaning & Visualization"
+author: "Kristina Riemer, University of Arizona"
+output: github_document
+urlcolor: blue
+---
+
+### Overall summary
+
+We can definitely use one experiment's data (the second one), which we have already started to do. We might be able to use experiments 4 and 5, but they only have two harvest dates (do we need six harvest dates?). Some experiments (1, 3, 6) have only one harvest date for each unique treatment. And the last experiment definitely cannot be used currently because there's no dry biomass measurements. 
+
+### Download data
+
+Data downloaded from [Google Sheet](https://docs.google.com/spreadsheets/d/134qzz1mcfKyGSS4vMOh0CONUECUuMiDyp6D_pzGjfi0/edit#gid=1249864874) as an .xlsx. 
+
+```{r}
+library(readxl)
+
+data_path <- "biocro_biomass_clean_darpa_files/Darpa_setaria_chambers_experiments.xlsx"
+sheets_names <- excel_sheets(data_path)
+
+biomass_sheets <- c(11, 9, 7, 5, 3, 2, 1)
+for(sheet in biomass_sheets){
+  exp_name <- paste0("exp", substr(sheets_names[sheet], 0, 1))
+  assign(exp_name, read_excel(data_path, sheet = sheets_names[sheet]))
+}
+```
+
+### First experiment
+
+Plots for first experiment. Plot by treatment started date (2), genotype (1), temperature (3), light intensity (3), separate plots by sowing date (2). Skipping conversion to mass by area. Removing yield for now. 
```

Comment:
I am  thinking that "Plots for first experiment" is confusing since experiment 1 appears to have been ruled out above?
Perhaps change "Skipping conversion to..." and "Removing yield..." to "We are not converting mass to area, or calculating yield at this time", or something similar?